lib/connection.js

```diff
@@ -209,16 +221,21 @@ class Connection extends EventEmitter {
 
   // Create a JSTP packet
   //   kind - packet kind
-  //   target - name of an interface or an application (optional)
+  //   target - value or array (will be flattened)
+  //            arguments for kind key, usually a name
+  //            of an interface or an application (optional)
   //   verb - action specific for different packet kinds
   //   args - action arguments
   //
   _createPacket(kind, target, verb, args) {
-    const packet = {
-      [kind]: (
-        target ? [this._nextPacketId, target] : [this._nextPacketId]
-      )
-    };
+    const packet = {};
+    if (Array.isArray(target)) {
```

Comment:
@aqrln, to avoid creation of an array with one element in some cases, it should probably be better for performance.